For_Andrey/task_3.py

```diff
@@ -0,0 +1,175 @@
+import random
+from math import ceil
+# Использовал math в методе number_of_car_refueling для округления до болшего числа. Можно было и без него.
+# Если модули лишний раз лучше не импортировать - дай знать.
+
+
+class Car:
+
+    New_car_cost = 10000
+    Repair_for_diesel_car = 700
+    Repair_for_petrol_car = 500
+    Mileage_before_repair_diesel = 150_000
+    Mileage_before_repair_petrol = 100_000
+    Diesel_fuel_cost = 1.8
+    Petrol_fuel_cost = 2.4
+    Fuel_consumption_diesel_on_hundred_km = 6 / 100
```

Comment:
Could you use 0.06 instead?

Reply:
I don't see difference
If you want to change it, will be easier